stage1/init/init.go

```diff
@@ -581,7 +582,12 @@ func stage1() int {
 		return 1
 	}
 
-	args, env, err := getArgsEnv(p, flavor, debug, n, insecureOptions)
+	canMachinedRegister := false
+	if flavor != "kvm" {
```

Comment:
fly neither writes a file nor does machined registration (this `init.go` code is not called for it and there is no analogue in the `stage1_fly/run/run.go` code).